buildscripts/patches/gl4es-mipmap-fix.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/src/gl/texture_compressed.c b/src/gl/texture_compressed.c
+index 4101519..6b8c8f8 100755
+--- a/src/gl/texture_compressed.c
++++ b/src/gl/texture_compressed.c
+@@ -214,11 +214,11 @@ void gl4es_glCompressedTexImage2D(GLenum target, GLint level, GLenum internalfor
+                 #endif
+             }
+         }
+-        if(level && bound->valid) {
++/*        if(level && bound->valid) {
```

Comment:
What does this do, why cannot this be fixed in gl4es?

Reply:
its not needed anymore(i hope, still compiling)

Reply:
So its not needed anymore, removed that.